activestorage/test/test_helper.rb

```diff
@@ -166,6 +174,12 @@ class User < ActiveRecord::Base
   def should_preprocessed?
     name == "transform via method"
   end
+
+  def thumb_size
+    return [200, 200] if name == "John"
+
+    [100, 100]
+  end
```

Comment:
@xjunior this method doesn't belong here. maybe you need it closer to where you use it? this is a global `active_storage` test helper, and i wonder how `thumb_size` that depends on a `name` being `John` and only used in your test case belongs here.